renovate-config.js

```diff
@@ -0,0 +1,42 @@
+const excludeList = [
+    "dotnet-sdk", // The dotnet SDK update is a non-trivial piece of work.
+];
+
+module.exports = {
+
+    timezone: "Australia/Brisbane",
+    requireConfig: "optional",
+    onboarding: false,
+
+    ignoreDeps: excludeList,
+    enabledManagers: ["nuget"],
+
+    // Full list of built-in presets: https://docs.renovatebot.com/presets-default/
+    extends: [
+        "config:base",
+        "group:monorepos",
+        "group:recommended",
+		":rebaseStalePrs",
+        ":automergeRequireAllStatusChecks",
+    ],
+
+    // Renovate will create a new issue in the repository.
+    // This issue has a "dashboard" where you can get an overview of the status of all updates.
+    // https://docs.renovatebot.com/key-concepts/dashboard/
+    dependencyDashboard: true,
+    dependencyDashboardTitle: "Tentacle Dependency Dashboard",
+
+    platform: "github",
+    repositories: ["OctopusDeploy/OctopusTentacle"],
+    reviewers: ["raynhamdev"],
+    labels: ["dependencies", "Tentacle"],
+    branchPrefix: "renovate-dotnet/",
+
+    // Limit the amount of PRs created
+    prConcurrentLimit: 10,
```

Comment:
This feels really high, shall we set it to something like 2 to start? I don't want us to have a lot of them open and feel pressured to merge a bunch at once, or get overwhelmed by the volume.